### DISCUSSION TITLE
Klangfarts heads up

### DISCUSSION THREAD

**Discussion by psiha** [jfalcou/kiwaku#60] [General]
Just a heads up on Clang's codegen issues I've come up so far - so that you can be aware of those and design around them when possible/appropriate :D

https://github.com/llvm/llvm-project/issues/58899
https://github.com/llvm/llvm-project/issues/58790
https://github.com/llvm/llvm-project/issues/58789
https://github.com/llvm/llvm-project/issues/52691

**Comment by jfalcou** [comment]
OK, let's keep those infos in the Discussion thread :)

**Comment by psiha** [comment]
lol didn't know there was such a thing on github :D
